gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -245,7 +247,9 @@ func (p *glusterBlockProvisioner) createSecretRef(nameSpace string, secretName s
 		Type: chapType,
 	}
 
+	p.volConfig.BlockSecret = secretName
 	secretRef := &v1.LocalObjectReference{}
+	p.volConfig.BlockSecretNs = nameSpace
 	if secret != nil {
 		_, err = p.client.Core().Secrets(nameSpace).Create(secret)
```

Comment:
Correction: I mean if provisioning pvs for multiple pvcs *in the same namespace*. Same idea though.